### PR TITLE
Added `muted` context manager for temproary switching signal off

### PR DESCRIPTION
**Pull request by pgjones**
This is useful whilst testing to remove a signal's affects.

### CODE REVIEW

File: src/blinker/base.py [hunk 1]
```diff
@@ -209,6 +210,19 @@ def connected_to(self, receiver, sender=ANY):
         else:
             self.disconnect(receiver)
 
+    @contextmanager
+    def muted(self):
```
Comment (Secrus) [inline src/blinker/base.py:214]: Could you please add this to the docs as well?

**Review by Secrus — CHANGES_REQUESTED** [review]

File: src/blinker/base.py (hunk 1 above)
Comment (pgjones) [reply src/blinker/base.py:214]: Yep, added some simple docs.

File: docs/index.rst [hunk 2]
```diff
@@ -157,6 +157,19 @@ value``) pairs:
   [(<function receive_data at 0x...>, 'received!')]
 
 
+Muting signals
+--------------
+
+To mute a signal, as may be required when testing, the
+:meth:`~Signal.mute` can be used as a context decorator:
```
Comment (Secrus) [inline docs/index.rst:164]: I think you meant `muted`

Comment (Secrus) [reply docs/index.rst:164]: On the other hand, `mute()` seems like a better choice for the method name. I will leave it up to you to choose one version.

**Review by Secrus — APPROVED** [review]

**Pull request merged**